pinot-spi/src/main/java/org/apache/pinot/spi/utils/GroovyTemplateUtils.java

```diff
@@ -77,5 +77,6 @@ public static String renderTemplate(String template)
 
   static {
     DATE_FORMAT.setTimeZone(TimeZone.getTimeZone("UTC"));
+    ((SimpleTemplateEngine) GROOVY_TEMPLATE_ENGINE).setEscapeBackslash(true);
```

Comment:
(nit) We can change the declaration of `GROOVY_TEMPLATE_ENGINE` to `SimpleTemplateEngine` to avoid this type cast for clarity

Reply:
agreed